### PR TITLE
WDP200802-10

### PR DESCRIPTION
**Pull request by Bosik-R**
to do:
change the ProductSearch select element to choose categories 

done:
i changed the select into a div with hover effect that unfolds down a list with links
i add a gradient effect on link hover

### CODE REVIEW

File: src/components/features/ProductSearch/ProductSearch.module.scss [hunk 1]
```diff
@@ -24,14 +24,69 @@
...
+        height: 45px;
+        font-size: 14px;
+        padding: 10px 20px;
+        color: $text-color;
+        transition: .5s ease-in-out;
+       
+        &:hover {
+          background-color: $primary;
+          color: #ffff;
+          padding-left: 80px;
```
Comment (Varenthein) [inline src/components/features/ProductSearch/ProductSearch.module.scss:57]: You store most of the colors as SCSS variables, so why have you left this one? ;)

Comment (Bosik-R) [reply src/components/features/ProductSearch/ProductSearch.module.scss:57]: didn't wont to interfere in the other task that handles the color variables in the settings.scss. I will correct it.

Comment (Bosik-R) [reply src/components/features/ProductSearch/ProductSearch.module.scss:57]: I add a new variable in the settings.scss and used it in ProductSearch component. have a nice Saturday :)

**Comment by Varenthein** [comment]
Just fix the conflicts and we're good to go :)

**Pull request merged**